dna/metrics.py

```diff
@@ -4,7 +4,7 @@
 import scipy.stats
 from sklearn.metrics import accuracy_score, mean_squared_error
 
-import utils
+from dna import utils
```

Comment:
This is what will cause the regular DNA to fail, but without it the testsuite will fail.